src/app/shared/module/poe/component/item-frame-header/item-frame-header.component.html

```diff
@@ -5,12 +5,12 @@
     </app-item-frame-query>
   </div>
   <div class="name" *ngIf="item.typeId">
-    <app-item-frame-query
-      [(property)]="queryItem.typeId"
-      [value]="item.typeId"
-      [disabled]="!queryable"
-    >
-      <span>{{ item.typeId | baseItemType: language }}</span>
+    <app-item-frame-query [(property)]="queryItem.typeId" [value]="item.typeId" [disabled]="!queryable">
+      <ng-container [ngSwitch]="getNameSpanNum(item)">
```

Comment:
not super important, but it might be nice if this function name was more descriptive

Reply:
Renamed it in 0cac5edbfb2f5c7687f7c18a1debfe0c8a7f24f8